mysql-test/suite/galera/t/MDEV-10715.test

```diff
@@ -0,0 +1,18 @@
+--source include/galera_cluster.inc
+--source include/have_innodb.inc
+
+--connection node_1
+create table t1(a int);
+set @@wsrep_gtid_seq_no=22;
+insert into t1 values(1);
```

Comment:
Maybe add one more insert , so that we can see cluster gtid seq is updated

Reply:
But it will be better to see that gtid seq no is incremented

Reply:
Added one more INSERT